test/passing/js_source.ml.ref

```diff
@@ -9793,3 +9790,28 @@ let _ =
         (match e with
         | x -> x)]
 ;;
+
+let f
+    ~first_arg
+    ~x
+    ~(* this is a comment about x *) last_arg
+    ~(* last argument *) last_arg
+    ~(* last argument *) last_arg (* last argument *)
+  =
+  assert false
+;;
+
+let f
+    ~first_arg
+    ~x
+    ~(* this is a comment about x *) last_arg
+    ~(* last argument *) last_arg (* last argument *)
+  =
+  assert false
+;;
+
+let f ~first_arg ~x ~(* this is a comment about x *) last_arg (* last argument *) =
+  assert false
+;;
+
+let f ~first_arg ~x ~(* this is a comment about x *) last_arg = assert false
```

Comment:
All this format exactly the same on master.

Reply:
Damn you are right, I'm curious what PR fixed this. I'm closing this one.